src/extension.ts

```diff
@@ -18,9 +18,7 @@ interface Params {
     file: string | null
 }
 
-const DECORATION_TYPE = sourcegraph.app.createDecorationType()
-const SETTINGSCONFIG = resolveSettings(sourcegraph.configuration.get<Settings>().value)
-const SENTRYORGANIZATION = SETTINGSCONFIG['sentry.organization']
+const SENTRYORGANIZATION = resolveSettings(sourcegraph.configuration.get<Settings>().value)['sentry.organization']
```

Comment:
Do you still need this?

Reply:
Yes, for `decorateLine()` and `buildUrl`.